system/jlib/jtrace.cpp

```diff
@@ -808,8 +865,11 @@ class CSpan : public CInterfaceOf<ISpan>
 
     ISpan * createClientSpan(const char * name, const SpanTimeStamp * spanStartTimeStamp = nullptr) override;
     ISpan * createInternalSpan(const char * name, const SpanTimeStamp * spanStartTimeStamp = nullptr) override;
+    ISpan * createConditionalInternalSpan(const char * name, stat_type thresholdMs) override;
+    ISpan * createConditionalClientSpan(const char * name, stat_type thresholdMs) override;
 
-    virtual void endSpan() final override
+    //virtual void endSpan() final override
+    virtual void endSpan() override
```

Comment:
Initial though was based on avoiding the endspan call for spans which did not meet the duration threshold, however, initialized otel spans are exported regardless.